__tests__/pages/hc/index.test.tsx

```diff
@@ -35,8 +35,8 @@ it("redirects to the loading page when online", async () => {
 
   expect(routerReplaceMock).toHaveBeenCalledTimes(1);
   expect(routerReplaceMock).toHaveBeenCalledWith(
-    { pathname: "/hc/[processRef]/loading" },
-    { pathname: "/hc/test-process-ref/loading" }
+    { pathname: "/thc/[processRef]/loading" },
+    { pathname: "/thc/test-process-ref/loading" }
   );
 });
 
```

Comment:
should this be thc?

Reply:
those will be changed in one of the subsequent tickets